dataline-webapp/src/pages/SourcesPage/pages/AllSourcesPage/components/LastSyncCell.tsx

```diff
@@ -20,14 +20,20 @@ type IProps = {
   enabled?: boolean;
 };
 
-const LastSyncCell: React.FC<IProps> = ({ value, enabled }) => (
-  <Content enabled={enabled}>
-    <CalendarIcon icon={faCalendarAlt} />
-    <FormattedRelativeTime
-      value={(value - Date.now()) / 1000}
-      updateIntervalInSeconds={60}
-    />
-  </Content>
-);
+const LastSyncCell: React.FC<IProps> = ({ value, enabled }) => {
+  if (!value) {
+    return null;
+  }
+
+  return (
+    <Content enabled={enabled}>
+      <CalendarIcon icon={faCalendarAlt} />
+      <FormattedRelativeTime
+        value={(value - Date.now()) / 1000}
```

Comment:
should this be `(Date.now() - value) / 1000` or does it take the absolute value somewhere?

Reply:
We use `FormattedRelativeTime`. It forms message for time period. If value is negative user will see message like `5 min ago`. If value is positive message will be `in 5 min`